client/js/components/Avatar/Avatar.jsx

```diff
@@ -1,22 +1,15 @@
 import React from "react";
 import "./Avatar.less";
 
-export default React.createClass( {
-	propTypes: {
-		className: React.PropTypes.string,
-		owner: React.PropTypes.string,
-		size: React.PropTypes.number
-	},
-	getDefaultProps() {
-		return {
-			owner: "anonymous",
-			size: 16,
-			className: "avatar"
-		};
-	},
-	render() {
-		return (
-			<img className={ this.props.className } src={ `https://avatars.githubusercontent.com/${ this.props.owner }?s=${ this.props.size }` } />
-		);
-	}
-} );
+const Avatar = ( { className = "avatar", owner = "anonymous", size = 16 } ) => {
```

Comment:
I think I prefer what you did (destructuring with default values) rather than using `Avatar.defaultProps`, however, if the signature got too long it might feel unwieldy? In that case, I guess the signature could be broken into multiple lines or maybe then `.defaultProps` would be preferred? This might be good to talk about further regarding a best practice or guideline... unless it is already in the wiki I've heard so much about, but have never seen yet.